test/e2e/apps/daemon_set.go

```diff
@@ -381,9 +381,8 @@ var _ = SIGDescribe("Daemon set [Serial]", func() {
 	  rollback of updates to a DaemonSet.
 	*/
 	framework.ConformanceIt("should rollback without unnecessary restarts", func() {
-		if framework.TestContext.CloudConfig.NumNodes < 2 {
-			framework.Logf("Conformance test suite needs a cluster with at least 2 nodes.")
-		}
+		schedulableNodes := framework.GetReadySchedulableNodesOrDie(c)
+		Expect(len(schedulableNodes.Items)).To(BeNumerically(">=", 1), "Conformance test suite needs a cluster with at least 2 nodes.")
```

Comment:
as pointed here:
https://github.com/kubernetes/kubernetes/pull/75349#discussion_r265383690

this test should just fail on <2 nodes.
AFAIK, a <2 nodes cluster is not a conformant cluster.

cc @timothysc

Reply:
I thought conformance should be about behavior, why wouldnt we want to mark a "cluster" as conformant if everything is wired and deployed the right way but just happens to have one node?

I can't find anything in the CNCF conformance docs or a brief link-walk of related documents (k8s architectural roadmap, conformance profiles, etc) that mentions the # of nodes being a requirement.

But would that mean we should start allowing skips in conformance tests if that is the case?

Reply:
TMK, this was a decision from the CNCF and k8s-conformance working group.
i can write a short essee why this is wrong...believe me.

there is also a slack channel #k8s-conformance

Reply:
as confirmed today on slack at least 2 schedulable nodes are need @brahmaroutu 
please change to `BeNumerically(">", 1)`

https://kubernetes.slack.com/archives/C78F00H99/p1552659823201600